.golangci.yml

```diff
@@ -81,3 +81,4 @@ linters:
     - unconvert
     - unparam
     - unused
+
```

Comment:
Is the solution just add an space?

Reply:
But the CI will call the github action not the makefile target command.  So, how would you like to validate it with the PR/ci? 

Anyway, I re-run the command now:

```
 $ make lint
golangci/golangci-lint info checking GitHub for tag 'v1.51.2'
golangci/golangci-lint info found version: 1.51.2 for v1.51.2/darwin/arm64
golangci/golangci-lint info installed /Users/camiladeomacedo/go/src/sigs.k8s.io/kubebuilder/bin/golangci-lint
/Users/camiladeomacedo/go/src/sigs.k8s.io/kubebuilder/bin/golangci-lint run
```

And yes, the warnings is no longer faced. So, we can close this one and the issue.
Thank you a lot for your contribution and collaboration.